src/xunit.runner.visualstudio/VsTestRunner.cs

```diff
@@ -23,6 +24,9 @@ namespace Xunit.Runner.VisualStudio
     [FileExtension(".exe")]
     [DefaultExecutorUri(Constants.ExecutorUri)]
     [ExtensionUri(Constants.ExecutorUri)]
+#if !WINDOWS_UAP
+    [Category("managed")]
```

Comment:
What does the UWP runner do instead?

Reply:
UWP or NetCore runner don't really have an issue, since for both of them platfrom strictly picks adapter from source location only.
The main issue is with fullclr where we have various locations we can get adapter from, hence we end up giving source dll's to adapters, which we can avoid.